### ISSUE TITLE
add martingale 

### ISSUE THREAD

**Issue by lordmen99** [collinpowell/bybit-bot-server#1]
nice bot bro, btw. can u add martingale system? so, when not meet good signal (floating) will be marti with 1-2 multiplier.

**Comment by collinpowell** [comment]
Give more explanation sir

**Comment by lordmen99** [comment]
let say, coming sell signal then in few minutes coming buy signal when get this sell will have floating / minus. let say -12% then bot will sell again with multi plier amount 1,2X from sell amount. then when trend down and get plus profit bot will close that. vice versa for buy signal

**Comment by collinpowell** [comment]
So how do I implement this

**Comment by collinpowell** [comment]
Margin gale

**Comment by lordmen99** [comment]
> let say, coming sell signal then in few minutes coming buy signal when get this sell will have floating / minus. let say -12% then bot will sell again with multi plier amount 1,2X from sell amount. then when trend down and get plus profit bot will close that. vice versa for buy signal

this for the flow trade bro

**Comment by collinpowell** [comment]
I taught of that, still figuring how to fix,

But can just program one trade at a time

**Comment by lordmen99** [comment]
> I taught of that, still figuring how to fix,
> 
> But can just program one trade at a time

try change trade flow like this bro, example for signal try add can get from tradingview then let say get sell signal then its in floating minus 15% then doing sell again with 1.2 amount first sell. its until 3-5 orders cycle. then if get profit close all

**Comment by collinpowell** [comment]
> > I taught of that, still figuring how to fix,
> > But can just program one trade at a time
> 
> try change trade flow like this bro, example for signal try add can get from tradingview then let say get sell signal then its in floating minus 15% then doing sell again with 1.2 amount first sell. its until 3-5 orders cycle. then if get profit close all

Okay sir, just read about margin gale would implement when chanced, did you test bot????

**Comment by lordmen99** [comment]
> > > I taught of that, still figuring how to fix,
> > > But can just program one trade at a time
> > 
> > 
> > try change trade flow like this bro, example for signal try add can get from tradingview then let say get sell signal then its in floating minus 15% then doing sell again with 1.2 amount first sell. its until 3-5 orders cycle. then if get profit close all
> 
> Okay sir, just read about margin gale would implement when chanced, did you test bot????

yes. its nice signal but need add tradingview webhook signal will be more awesome
